controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -362,6 +367,20 @@ private File determineMainFile(final String rawName, final String name) {
         throw ControllerLogger.ROOT_LOGGER.mainFileNotFound(name != null ? name : rawName, configurationDir);
     }
 
+    private String translateFileAlias(String alias) {
+        final int pos = rawFileName.lastIndexOf('.');
+        final String base = pos != -1 ? rawFileName.substring(0, pos) : rawFileName;
+        switch (alias) {
```

Comment:
Do we really want a list like this in the code, this becomes something else that we need to remember to maintain if the list of files changes.

If the argument does not specify a suffix could we not instead assume standalone-{argument}.xml and check if we have a match?

Reply:
I mean we could do that as well. The intention is not having to type long config names and e.g. `load-balancer` or 'microprofile` are IMHO still kinda long.